vic/drivers/python/setup.py

```diff
@@ -86,6 +88,94 @@ def write_version_py(filename=None):
 
 
 # -------------------------------------------------------------------- #
+def maybe_eval_string(s):
```

Comment:
I would make this function name a little more informative, e.g. `eval_string_bw_brackets`